nixos/modules/services/networking/wireguard.nix

```diff
@@ -320,7 +335,16 @@ let
                 # cannot be used with systemd timers (see `man systemd.timer`),
                 # which is why `simple` with a loop is the best choice here.
                 # It also makes starting and stopping easiest.
+                #
+                # Retry even after "Name or service not known" dns failures:
+                Restart = "always";
+                RestartSec = "${toString (if null != peer.dynamicEndpointRefreshRestartSeconds
```

Comment:
`toString` is not needed.

Reply:
@jian-lin I think toString is required, the options' values are integers and if you don't cast explicitly you get `error: cannot coerce an integer to a string`

Reply:
Did you also remove the quotation?

Reply:
No I just removed the `toString` part, assigning the bare integer should work

Reply:
I got the same coercion error initially, but assigning the integer directly worked :-)